plugin_test.go

```diff
@@ -227,6 +227,83 @@ func TestMessageWithExtensionOnNestedField(t *testing.T) {
 		})
 }
 
+// Try a complex message with extension with a few edge cases
+func TestMessagesWithExtensionExceptions(t *testing.T) {
+	testConvert(t, `
```

Comment:
just nitting around, let's keep adding those `t.Parallel()` to detect data races

```suggestion
func TestMessagesWithExtensionExceptions(t *testing.T) {
	t.Parallel()
	testConvert(t, `
```